localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ 
+ saurok_religion:0 "Krosi"
+ saurok_religion_adj:0 "Krosi"
+ saurok_religion_adherent:0 "Krosi"
+ saurok_religion_adherent_plural:0 "Krosi"
+ saurok_religion_desc:0 "A faith in Kros, the invisible devilsaur in the sky who created lizard races and jungles of Azeroth. Azeroth's devilsaurs are considered his direct offsprings and worshipped as minor deities. $saurok_religion$ preaches to never stop fighting, never stop growing. Each day, stronger than before."
+ 
+ taunka_religion:0 "Roanaki"
+ taunka_religion_adj:0 "Roanaki"
+ taunka_religion_adherent:0 "Roanaki"
+ taunka_religion_adherent_plural:0 "Roanaki"
+ taunka_religion_desc:0 "A belief system originating in the icy deserts of Northrend. Unless other $shamanism_group_adherent_plural$, $taunka_religion_adherent_plural$ force the land and the elements to yield to their will. Their relationship with nature has been one of constant struggle and grim perseverance."
+ 
+ ape_group:0 "Apism"
+ ape_group_adj:0 "Apish"
+ ape_group_adherent:0 "Apist"
+ ape_group_adherent_plural:0 "Apists"
+ ape_group_desc:0 "A group of faiths founded by the hozen. Mostly consists of shamanic practices and promotes a simple life for pleasure."
+    
+ lizard_group:0 "Reptilism"
+ lizard_group_adj:0 "Reptilian"
+ lizard_group_adherent:0 "Reptilist"
+ lizard_group_adherent_plural:0 "Reptilists"
+ lizard_group_desc:0 "A group of faiths founded by the saurok. Mostly consists of shamanic practices and praises war, warrior lifestyle and cold blood."
+ 
+ mystery_of_the_makers:0 "Pantheonism"
+ mystery_of_the_makers_adj:0 "Pantheonic"
+ mystery_of_the_makers_adherent:0 "Pantheonist"
+ mystery_of_the_makers_adherent_plural:0 "Pantheonists"
+ mystery_of_the_makers_desc:0 "An original branch of $titanic_group_adj$ religions. They worship the Pantheon led by Aman'Thul and see themselves as the only guardians of Azeroth world soul. They consider their duty to defend titanic complexes all over the world until the Titans return."
+ 
+ thunder_king:0 "Shenism"
+ thunder_king_adj:0 "Shenic"
+ thunder_king_adherent:0 "Shenist"
+ thunder_king_adherent_plural:0 "Shenists"
+ thunder_king_desc:0 "Lei Shen, the Thunder King was a legendary mogu emperor and demigod that single-handedly defeated the despondent Titan Keeper, Ra-den within the Thundering Mountain and then subsequently stole both his power and a fraction of Aman'Thul's power, giving him immense mastery over the storms whose power could rival many of the cosmos' powerful beings, which he then used to conquer ancient Pandaria and forge an empire of the mogu, with their mandate being the continued work of the Makers after their supposed demise at the hands of the Dark Titan. Highly meritocratic, the followers of the Thunder King believe they are the only ones worthy to rule over and uphold the ancient titan legacy in Azeroth."
```

Major run on sentence. Changed the last sentence a bit as well
```suggestion
 thunder_king_desc:0 "Lei Shen, the Thunder King was a legendary mogu emperor and demigod that single-handedly defeated the despondent Titan Keeper, Ra-den within the Thundering Mountain. Lei Shen subsequently stole both Ra-den's power and a fraction of Aman'Thul's power, giving him immense mastery over the storms whose power could rival many of the cosmos' powerful beings. He proceeded to use his newfound to conquer ancient Pandaria and forge an empire of the mogu, with their mandate being the continued work of the Makers after their supposed demise at the hands of the Dark Titan. Highly meritocratic, the followers of the Thunder King believe they are the only ones worthy to rule and uphold the ancient titan legacy in Azeroth."
```